riscv-regs/src/inst.rs

```diff
@@ -2863,6 +2863,12 @@ pub const CSR_MHPMCOUNTER28H: u16 = 0xb9c;
 pub const CSR_MHPMCOUNTER29H: u16 = 0xb9d;
 pub const CSR_MHPMCOUNTER30H: u16 = 0xb9e;
 pub const CSR_MHPMCOUNTER31H: u16 = 0xb9f;
+pub const CSR_SMCOUNTEREN: u16 = 0xd00;
```

Comment:
Just leave a comment here saying these are temporary. The actual CSR number gets finalized after ARCH review